akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -270,6 +297,26 @@ class ClusterClient(settings: ClusterClientSettings) extends Actor with Stash wi
     else if (contacts.size == 1) (initialContactsSel ++ contacts) foreach { _ ! GetContacts }
     else contacts foreach { _ ! GetContacts }
   }
+
+  def buffer(msg: Any): Unit =
```

Comment:
Opening up such classes for extension makes it very difficult for us to change things (also in a binary compatible way). I actually made this class `final` for this reason.

I could do an exception for the ClusterClient, but then I will make everything private/final except `def buffer` and `def sendBuffered`.

and yes, then you can override them in Java